draft-homburg-deleg-incremental-deleg.md

```diff
@@ -0,0 +1,621 @@
+For example, if the delegating zone is `example.`, then a delegation to subzone `customer.example.` is realized by an SVCB RRset at the name `customer._deleg.example.` in the parent zone.
+A fully scoped delegating name (such as `customer._deleg.example.`) is referred to further in this document as the "delegation point".
+
+The use of the SVCB RR type requires a mapping document for each service type.
+This document uses the SVCB for the "dns" service type and the contents of the SVCB SvcParams MUST be interpreted as specified in Service Binding Mapping for DNS Servers {{!RFC9461}}.
+At the delegation point (for example `customer._deleg.example.`), the host names of the authoritative name servers for the subzone, are given in the TargetName RDATA field of SVCB records in ServiceMode.
+Port Prefix Naming {{Section 3 of RFC9461}} is not used at the delegation point, but MUST be used when resolving the aliased to name servers with "dns" service type SVCB RRs in AliasMode.
+
+Note that if the delegation is outsourcing to a single operator represented in a single SVCB RRset, it is RECOMMENDED to refer to the name of the operator's SVCB RRset with a CNAME on the delegation point instead of an SVCB RR in AliasMode {{Section 10.2 of !RFC9460}}.
+
+## Examples
+
+### One name server within the subzone
+
+~~~~
+$ORIGIN example.
+@                  IN  SOA   ns zonemaster ...
+customer1._deleg   IN  SVCB  1 ( ns.customer1
+                                 ipv4hint=198.51.100.1,203.0.113.1
+                                 ipv6hint=2001:db8:1::1,2001:db8:2::1
+                               )
+~~~
+{: #zone-within title="One name server within the subzone"}
+
+### Two name servers within the subzone
+
+    $ORIGIN example.
+    @                  IN  SOA   ns zonemaster ...
+    customer2._deleg   IN  SVCB  1 ns1.customer2 ( ipv4hint=198.51.100.1
+                                                   ipv6hint=2001:db8:1::1
+                                                 )
+                       IN  SVCB  1 ns2.customer2 ( ipv4hint=203.0.113.1
+                                                   ipv6hint=2001:db8:2::1
+                                                 )
+{: #zones-within title="Two name servers within the subzone"}
+
+### Outsourced to an operator
+
+    $ORIGIN example.
+    @                  IN  SOA   ns zonemaster ...
+    customer3._deleg   IN  CNAME _dns.ns.operator1
+{: #outsourced-cname title="Outsourced with CNAME"}
+
+Instead of using CNAME, the outsourcing could also been accomplished with an SVCB RRset with a single SVCB RR in AliasMode.
+The configuration below is operationally equivalent to the CNAME configuration above.
+It is RECOMMENDED to use a CNAME over an SVCB RRset with a single SVCB RR in AliasMode ({{Section 10.2 of !RFC9460}}).
```

Comment:
Maybe a comment is needed that with CNAME you need _dns and with an AliasMode SVCB you don't.